doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -44,7 +45,27 @@ Components:
 Each Teuthology test *run* contains multiple test *jobs*. Each job runs in an 
 environment isolated from other jobs, on a different collection of test nodes.
 
-To test a change in Ceph, follow these steps:
+Workflow Overview
+*****************
+
+.. image:: workflow.png
+
+
+To test a change in ceph, start by pushing a branch with your changes to 
+`ceph-ci`_ repository. This will automatically trigger the Jenkins process 
+to build ceph binaries - the status of the build can be observed on `Shaman`_. 
+These built packages will be uploaded on `Chacra`_. 
+
+To schedule a teuthology integration test against this new build, you will 
```

Comment:
s/teuthology/Teuthology/

Reply:
and s/teuthology/Teuthology/ in all other uses of the string when they refer to the integration test framework itself and not any of the commands that are a part of it.